webdriver/tests/bidi/browsing_context/capture_screenshot/format.py

```diff
@@ -0,0 +1,24 @@
+import pytest
+
+
+@pytest.mark.asyncio
+async def test_format(bidi_session, top_context, inline):
+    await bidi_session.browsing_context.navigate(
+        context=top_context["context"],
+        url=inline("<div style='margin-top:2000px'>foo</div>"),
+        wait="complete")
+
+    png_screenshot = await bidi_session.browsing_context.capture_screenshot(
+        context=top_context["context"], format={"type": "image/png"})
+    jpeg_screenshot = await bidi_session.browsing_context.capture_screenshot(
+        context=top_context["context"], format={"type": "image/jpeg"})
+    jpeg_quality_screenshot = await bidi_session.browsing_context.capture_screenshot(
+        context=top_context["context"],
+        format={
+            "type": "image/jpeg",
```

Comment:
There is only one other image type checked here. Which other types should be supported by default?

Also can we somewhat test different quality settings? I assume we could verify by just comparing the size of the base64 encoded data?

Reply:
The only image type required to be supported in the HTML standard is `image/png` (https://html.spec.whatwg.org/multipage/canvas.html#a-serialisation-of-the-bitmap-as-a-file)

I've added a test for quality.